plugins/vestige/vestige.cpp

```diff
@@ -160,6 +164,10 @@ vestigeInstrument::vestigeInstrument( InstrumentTrack * _instrument_track ) :
 			 Qt::QueuedConnection );
 }
 
+void vestigeInstrument::bindView(VestigeInstrumentView * view)
+{
+	m_pView = view;
```

Comment:
Can you use Qt's signal-slot connection instead of this approach? It's considered bad to store the reference to the view in the model.

Reply:
Done